assets/css/app.scss

```diff
@@ -11,6 +11,8 @@
   --tw-content: '' !important;
 }
 
+@import "../node_modules/@mbta/rider-design-system/dist/variables.light.css" (prefers-color-scheme: light);
+@import "../node_modules/@mbta/rider-design-system/dist/variables.dark.css" (prefers-color-scheme: dark);
```

Comment:
because the button styles (and in the future, other component styles) are defined using these CSS variables, we get light and dark mode versions this easily!